assets/components/ConnectionForm.svelte

```diff
@@ -110,10 +112,14 @@ async function submitForm(e) {
     </p>
   </TextField>
 
-  <TextField name="nick" placeholder="{l('Ex: your-name')}">
+  <TextField name="nick" placeholder="{l('Ex: superman')}">
     <span slot="label">{l('Nickname')}</span>
   </TextField>
 
+  <TextField name="realname" placeholder="{l('Ex: Clark Kent')}">
+    <span slot="label">{l('Real name')}</span>
```

Comment:
Is it less or more confusing with "Your name"? Also, do we need some help text to explain that this will be visible to other users you chat with?

Reply:
I think "superman/Clark Kent" is fine, but "nick/Your name" would be fine too. No strong preference.

How about "Real name (visible in WHOIS)"?

Reply:
Changed as requested.